### ISSUE TITLE
Keyboard is popping up everytime user selects a contact when creating a new group

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#20636]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Click on FAB
2. Click on "New group"
3. Select a contact
4. Hide the keyboard
5. Select another contact

## Expected Result: 
Keyboard should not popup every time user selects a new contact as this behavior is not seen on Android app
## Actual Result: 
Keyboard pops up every time user selects a new contact

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [x] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.26-3
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/43996225/5e5fd7d2-f3fd-462c-a4c9-60fac8f186f7

https://github.com/Expensify/App/assets/43996225/b64abd64-61c8-4a74-b096-da025608caf9

**Expensify/Expensify Issue URL:**
**Issue reported by:** @nathan-mulugeta
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1686153010639509

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by joekaufmanexpensify** [comment]
Reproduced on Android mobile web. Also confirmed we don't do this on android native, or iOS native. I agree we should not open the keyboard every time a new contact is selected. 

https://github.com/Expensify/App/assets/54866469/ba273fce-a413-4e25-87df-c0bd5fd98ced

**Comment by Santhosh-Sellavel** [comment]
@huzaifa-99 you mean `!Browser.isMobile()` right?

**Comment by Santhosh-Sellavel** [comment]
@marcaaron @huzaifa-99's [proposal](https://github.com/Expensify/App/issues/20636#issuecomment-1588400315) looks good to me.
🎀 👀 🎀 
C+ Reviewed

**Comment by marcaaron** [comment]
>if we want to keep the focus on the input field on desktop web but not on mobile web, then we can only add shouldFocusOnSelectRow for mobile platform and when we are creating a group chat, something like this

I think we do. If that's the current behavior we should not change it. So that would be the alternate proposal in this case?

@Santhosh-Sellavel you may have implied with your previous question that the alternative proposal is what we want - though, not sure. In the future, try make it clear whether you are recommending the main proposal or anything from the alternate proposal. Thank you 🙇

**Comment by Santhosh-Sellavel** [comment]
@joekaufmanexpensify It got merged within 3 biz days. Melvin is confused here.

**Comment by Natnael-Guchima** [comment]
@joekaufmanexpensify I have reported similar issue on invite members to workspace section. And i see the issue on the workspace section is fixed here. Am I eligible for reporting bonus. 

Here is my report: https://github.com/Expensify/App/issues/17193

**Comment by joekaufmanexpensify** [comment]
>It got merged within 3 biz days. Melvin is confused here.

Agreed. Assignment to issue occurred on 2023-06-13 at 6:49pm ET. PR was merged on 2023-06-16 at 2:55pm ET. This is ~4 hours less than the 72 hour threshold, so this qualifies for a speed bonus.

**Comment by joekaufmanexpensify** [comment]
@Natnael-Guchima I'd say no. We closed the issue you reported because we [were not able to reproduce it](https://github.com/Expensify/App/issues/17193#issuecomment-1520638565). Whereas the bug in this issue was consistently reproducible, which is why we took action here. 

Given the behavior you reported was not consistently reproducible, we wouldn't issue a reporting bonus for it.

**Comment by joekaufmanexpensify** [comment]
@huzaifa-99 offer sent for $1,000! ($500 will be issued as bonus)

**Comment by joekaufmanexpensify** [comment]
@Santhosh-Sellavel offer sent for $1,000! ($500 will be issued as bonus)

**Comment by joekaufmanexpensify** [comment]
@Nathan-Mulugeta offer sent for $250!

**Comment by Santhosh-Sellavel** [comment]
@joekaufmanexpensify I will get done with the checklist this week. Regarding payment I'm just waiting for the internal process, please check [here](https://expensify.slack.com/archives/C02NK2DQWUX/p1687451561563029?thread_ts=1686871517.189609&cid=C02NK2DQWUX)

**Comment by joekaufmanexpensify** [comment]
Sounds good, thanks!

**Comment by joekaufmanexpensify** [comment]
@Santhosh-Sellavel we are all set to test your payment. Could you please complete your portion of the BZ checklist, and then I'll post new steps for payment in Slack.

**Comment by Santhosh-Sellavel** [comment]
Pinged you on a Slack thread about payment. Caught up on other priority items, I'll complete the checklist this week in a day or two!

**Comment by joekaufmanexpensify** [comment]
Sounds good! As soon as the BZ checklist is complete, we'll issue the payments on this issue (besides yours, which is held for the new process).

**Comment by Santhosh-Sellavel** [comment]
> **BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:
> 
> * [x]  [@Santhosh-Sellavel] The PR that introduced the bug has been identified. Link to the PR: This was untouched for a long time https://github.com/Expensify/App/pull/7702
> * [x]  [@Santhosh-Sellavel] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment: https://github.com/Expensify/App/pull/7702/files#r1249819025
> * [x]  [@Santhosh-Sellavel] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion: This could be caught with our existing checklist so we are good here!
> * [x]  [@Santhosh-Sellavel] Determine if we should create a regression test for this bug. Not sure worth adding it
> * [x]  [@Santhosh-Sellavel] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.
> * [x]  [@joekaufmanexpensify] Link the GH issue for creating/updating the regression test once the above steps have been agreed upon:

cc: @marcaaron Let me know if you differ on any of the above

**Comment by Santhosh-Sellavel** [comment]
### Regression Steps
1. Go to the new chat page from the fab icon
2. In desktop web and native devices, verify if the search input is focused then it should remain focused when any member is selected.
3. In mWeb, verify that selecting a member should not focus the search input again thus fixing the "keyboard popping" bug
4. Follow steps 2 and 3 for the workspace invite member page.

👍  or 👎

**Comment by marcaaron** [comment]
>Verify that selecting a member is not focusing on the search input again.

I would rephrase this like:

>Verify if the search input is focused then it should lose focus when any member is selected.

But otherwise 👍

**Comment by Nathan-Mulugeta** [comment]
Hey guys payment has not been processed for this issue. @joekaufmanexpensify

**Comment by joekaufmanexpensify** [comment]
Thanks! I was OOO the past two days, and previously we were waiting for the BZ checklist to be completed. I will finish this up and issue payment today.

**Comment by joekaufmanexpensify** [comment]
BZ checklist is now complete! All set to issue payment.

**Comment by joekaufmanexpensify** [comment]
@huzaifa-99 $1,500 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
@Nathan-Mulugeta $250 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
Of course, and no worries at all!

**Comment by joekaufmanexpensify** [comment]
Still held on paying Santhosh

**Comment by anmurali** [comment]
Santhosh has been paid on New Dot
